refresh/fromswagger/swaggerize.js

```diff
@@ -184,10 +177,6 @@ function parseSwagger (api) {
     })
   } while (foundNewRef)
 
-  if (Object.keys(resources).length === 0) {
-    throw new Error('no resources')
-  }
-
```

Comment:
Can you explain briefly why this is okay to remove? Does the validator ensure this is never the case?